decidim-elections/lib/gem_overrides/graphlient/client.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Graphlient
```

Comment:
Where is this used? I can only find references to this from the `Gemfile.lock` files.

Reply:
This comes from [decidim-bulletin-board](https://github.com/decidim/decidim-bulletin-board/search?q=Graphlient), used only in `decidim-elections`.

Reply:
At the moment of the implementation there was a contract constraint and i could not interfere. This should be migrated to decidim-elections or directly in graphlient gem ( if not done already)

Reply:
@andreslucena @alecslupu Since the `graphlient` gem is a dependency for [Decidim Bulletin Board](https://github.com/decidim/decidim-bulletin-board), I think we should move this monkey patch to the Decidim Bulletin Board, publish a new release of it and then update the dependency version in Decidim itself.

Reply:
Actually it seems Graphlient 0.5.0 already supports Ruby 3.0:
https://github.com/ashkan18/graphlient/blob/master/CHANGELOG.md

So if we update that to Decidim Bulletin Board, it should be fine.

I'll make the PR there.

Reply:
Currently waiting for https://github.com/decidim/decidim-bulletin-board/pull/251 to be merged and released.

Reply:
The bulletin board PR is now ready, all tests are passing there.

Waiting for review and once merged, we need to release a new version of the `decidim-bulletin_board` gem and update it here once released.

Reply:
@alecslupu New versions of the bulletin board related gems are now published to RubyGems.

Can you bump these three to `0.23.0`:
https://github.com/decidim/decidim/blob/a15c91cd006fec147d3c4705173e3ab0b9296923/decidim-elections/decidim-elections.gemspec#L21-L23

After that, you should be able to remove this override.

I believe no other changes are needed for this PR regarding the bulletin board. I am handling the server upgrade in a separate PR (#9133) but it won't affect this PR.